web-console/src/entry.ts

```diff
@@ -30,11 +30,14 @@ import "./entry.scss";
 const container = document.getElementsByClassName('app-container')[0];
 if (!container) throw new Error('container not found');
 
+const consoleConfig: any = (window as any).consoleConfig;
 ReactDOM.render(
   React.createElement(
     ConsoleApplication,
     {
-      version: '0.0.1'
+      version: '0.15.0',
```

Comment:
Hmm, it would be very nice to find a more elegant way to sync versions from Druid's pom.xml to the web console, but I don't really know what that is right now (and not really necessary to address in this PR i think)

Reply:
yeah... to be honest this is dead code. The version here is never used, the console makes a `GET /status` query to get the version. Removing.